sdk/entra/ci.yml

```diff
@@ -0,0 +1,34 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/entra/functions-authentication-events/      
+      # eng/common code changes trigger template pipeline for basic checking.
+      - eng/common/
```

Comment:
```suggestion
```

Reply:
Please remove these two lines, we don't want to trigger this pipeline for core eng changes (we do for template, hence why it gets copied over).

Reply:
Done